packages/tools/devtools/devtools-view/src/components/SettingsView.tsx

```diff
@@ -117,7 +106,7 @@ export function SettingsView(): React.ReactElement {
 			<div className={styles.option}>
 				<label className={styles.label}>Select theme</label>
 				<Dropdown
-					value={themeReadableOption[themeInfo.name as ThemeName]}
+					value={themeInfo.name === "dark" ? "Dark" : themeInfo.name}
```

Comment:
This is probably because in ThemeHelper.ts we do [this](https://github.com/microsoft/FluidFramework/blob/d8db010073e26f696a3eddfd5fcb71c3aa9a4499/packages/tools/devtools/devtools-view/src/ThemeHelper.ts#L22-L42), right? Let's fix that in this PR too; that code should use the enum as well. With that update this should end up being just `value={themeInfo.name}`, I think.

Reply:
Yes. It's because the `getFluentUIThemeToUse` has `dark` (with lower-cased "d") as its `name`, and our `SettingsView` component takes "dark" as its initial value. 

I wasn't so sure of the scope of this PR and whether I should be fixing some of the codes from `ThemeHelper` 😃 .

Reply:
Maybe ideally we would make those changes (use the enum everywhere _today_) in a separate PR, but it's close enough to this one that I think it's ok to bundle together. I expect those fixes to be relatively minor and they contribute to make the actual change from this PR as clean as we can make it. If making those adjustments starts to get complicated, then we can split it into its own task; but if it'll be an hour or two, let's include it here.